CONTRIBUTING.md

```diff
@@ -25,11 +25,27 @@ Same rules apply here as for bug reports and feature requests. Plus:
 * We prefer atomic commits. Please read
   [this excellent blog post](https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/) for
   more information, including the rationale.
+  For larger changes addressing several packages consider splitting your pull request,
+  using a single commit for each package changed.
```

Comment:
Please don't join the gang of "let's break the line for every new sentence. :)

Reply:
Sorry, I am used to writing AsciiDoc (TM) which recommends just doing that. :-)

Reply:
It's a pretty commonly used tool AFAIK so it perpetuating this annoying practice that goes against the rules of the English grammar, makes me a bit sad. :(

Reply:
Anyway, please change it. Otherwise, I'll just have to do it myself after merging.

Reply:
Oops, my bad. 

I did _not_ intend to force my bad habits on you. I had made the change as requested but somehow manged not to add it to the commit ... :-/